lighting/models/product_source_line.py

```diff
@@ -380,6 +440,64 @@ def _compute_wattage_display(self):
         string="Source",
     )
 
+    invisible_fields = fields.Json(
+        compute="_compute_invisible_fields",
+    )
+
+    def _get_all_dependent_fields(self):
```

Comment:
To be like the others?
```suggestion
    def _get_dependent_all_fields(self):
```